articles/resources/relnotes.md

```diff
@@ -17,6 +17,18 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: Sept 29th, 2020*
+
+This release contains the following:
+
+- Performance improvements for simulator.
+- IntelliSense support is enabled when using Python as a driver.
+- Improved IQ# Python interoperability.
+- Implemented new gate for external contribution.
```

Comment:
- Improvements to the [`Microsoft.Quantum.Arrays` namespace](xref:microsoft.quantum.arrays) (https://github.com/microsoft/QuantumLibraries/issues/313)